packages/@expo/cli/src/start/server/metro/MetroBundlerDevServer.ts

```diff
@@ -935,8 +1034,20 @@ export class MetroBundlerDevServer extends BundlerDevServer {
   // Metro HMR
 
   private setupHmr(url: URL) {
-    debug('[CLI]: Register SSR HMR bundle URL:', url.toString());
-    // TODO: Pull in HMR SSR code.
+    const onReload = () => {
+      // Send reload command to client from Fast Refresh code.
+      debug('[SSR]: Reload requested.');
+
+      this.onReloadRscEvent?.();
+
+      this.broadcastMessage('sendDevCommand', {
+        name: 'rsc-reload',
+        // TODO: Target only certain platforms
+        // platform,
```

Comment:
Does this need to be resolved in this PR?

Reply:
it requires a much larger refactor, will hold off for now.